### PR TITLE
Fits rois

### PR DESCRIPTION
**Pull request by henrikef**
Added data_radius parameter back in for map ROI as it is needed by some of the map display functions.

### CODE REVIEW

**Comment by henrikef** [comment]
@cbrisboi or @colasri  could you review this? It's a small change.

File: hawc_hal/region_of_interest/healpix_map_roi.py [hunk 1]
```diff
@@ -31,6 +31,8 @@ def __init__(self, model_radius, roimap=None, roifile=None, threshold=0.5, *args
 
         :param model_radius: radius of the model cone. Either an astropy.Quantity instance, or a float, in which case it
         is assumed to be the radius in degrees
+        :param data_radius: radius used for displaying maps. Either an astropy.Quantity instance, or a float, in which case it
```
Comment (colasri) [inline hawc_hal/region_of_interest/healpix_map_roi.py:37]: Digression: The GitHub interface only allows me to comment on changed lines +/- 3 lines? This comments is for some code 4 and 9 lines above this one...

The `roi = HealpixMapROI(...)` examples above should be updated with the new keyword.

**Review by colasri — CHANGES_REQUESTED** [review]
I haven't run the test yet, I don't have a working version of 3ML at this time. I'll install it.

File: hawc_hal/region_of_interest/healpix_map_roi.py (hunk 1 above)
Comment (henrikef) [reply hawc_hal/region_of_interest/healpix_map_roi.py:37]: Done :)

**Comment by colasri** [comment]
I still don't have a test environment working, so IDK. @cbrisboi ?

**Comment by henrikef** [comment]
After setting `set_geminga_paper` to xfail (probably needs fixes to astromodels, see https://github.com/threeML/astromodels/pull/101 , the linux tests pass now. MacOS still fails (ROOT fails to import).

**Comment by colasri** [comment]
The Geminga test used to work. When did we break it?
Maybe we should slow down with the MR until the CI are back?

**Comment by henrikef** [comment]
It worked because the model parser in astromodels was broken and would drop some parameter links. That was fixed, see https://github.com/threeML/astromodels/pull/86#issuecomment-445961036 . Now it fails when Geminga's and Monogem's diffusion coefficients are linked, but we remove one of the sources from the model to calculate the TS. (I'd call that user error, actually).

In any case, I have a fix where astromodels will automatically unlink parameters in case of sources being removed, see https://github.com/threeML/astromodels/pull/101 . But, I need to write some unit tests and make sure it works with the current master of astromodels before merging. I did test locally and `test_geminga_paper` runs fine with my astromodels branch.

File: hawc_hal/region_of_interest/healpix_map_roi.py (hunk 1 above)
Comment (henrikef) [reply hawc_hal/region_of_interest/healpix_map_roi.py:37]: @colasri do you agree with these changes?

File: tests/test_geminga_paper.py [hunk 2]
```diff
@@ -14,7 +15,7 @@
 import argparse
 from collections import namedtuple
 
-
+@pytest.mark.xfail
```
Comment (colasri) [inline tests/test_geminga_paper.py]: Do we really need that? Did this PR break the test?

Comment (henrikef) [reply tests/test_geminga_paper.py]: Geminga test is broken due to a recent(ish) change in astromodels. See https://github.com/threeML/hawc_hal/pull/19#issuecomment-481512104

Comment (henrikef) [reply tests/test_geminga_paper.py]: After 1.5 years, I finally got around to fixing the Geminga test in another branch.

**Pull request merged**